src/ServiceInsight/Explorer/EndpointExplorer/EndpointExplorerViewModel.cs

```diff
@@ -170,9 +170,14 @@ public void RefreshData()
             }
 
             ServiceControlRoot.Children.Clear();
-            foreach (var endpoint in endpoints.OrderBy(e => e.Name))
+
+            var endpointInstancesGroupedByName = endpoints.GroupBy(x => x.Name);
+            foreach (var scaledOutEndpoint in endpointInstancesGroupedByName)
             {
-                ServiceControlRoot.Children.Add(new AuditEndpointExplorerItem(endpoint));
+                var instances = scaledOutEndpoint.ToList();
+                var hostNames = instances.Select(instance => instance.HostDisplayName);
+                var tooltip = string.Join(", ", hostNames);
```

Comment:
Have no preference and it wasn't specified how to separate between host names. Is stacking them would be better @johnsimons ?